### PR TITLE
Advisor Module Updates

### PR DESCRIPTION
**Pull request by Prasanna-Padmanabhan**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [x] Each command and parameter has a meaningful description.
- [x] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

**Comment by promptws** [comment]
View a preview at https://prompt.ws/r/Azure/azure-cli/5334
This is an **experimental preview** for @microsoft users.

File: src/command_modules/azure-cli-advisor/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+1.0.0
```
Comment (tjprescott) [inline src/command_modules/azure-cli-advisor/HISTORY.rst]: If this is a GA release, it should be 2.0.0

**Comment by tjprescott** [comment]
https://github.com/Azure/azure-cli/issues/5343

File: src/command_modules/azure-cli-advisor/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,10 @@
...
+++++++
+* GA release.
```
Comment (tjprescott) [inline src/command_modules/azure-cli-advisor/HISTORY.rst]: Here you will want to call out whatever breaking changes were made. For example
```
* BC: `advisor configuration set` renamed to `advisor configuration update`
```
and so on.

**Comment by promptws** [comment]
View a preview at https://prompt.ws/r/Azure/azure-cli/5334
This is an **experimental preview** for @microsoft users.

**Comment by tjprescott** [comment]
The needs to be rebased and missed the code-complete for Sprint 31.  Moving to Sprint 32.

**Pull request merged**